static/js/bundles/app.js

```diff
@@ -16,7 +16,7 @@ import "blueimp-md5/js/md5.js";
 import "clipboard/dist/clipboard.js";
 import "string.prototype.codepointat/codepointat.js";
 import "winchan/winchan.js";
-import "handlebars/dist/handlebars.runtime.js";
+import "handlebars/dist/cjs/handlebars.runtime.js";
```

Comment:
What's the story with this change?  Are those two both files in the module, or what?

Reply:
When importing the module without explicitly specifying the path, ie `import "handlebars/runtime";`, webpack actually resolve it to `handlebars/dist/cjs/handlebars.runtime.js`. Also after adding handlebars-loader, the original Handlebars from `handlebars/dist/handlebars.runtime.js` can't find the templates anymore for some reason.

Also is there a reason why we specify the exact file we import instead of just giving the name?

Reply:
In some cases, it makes a difference because `whatever/dist/whatever.js` adds itself to the global `window` namespace while `whatever` is a module that expects to be used the way modules are supposed to be used. We have lots of code that pulls things from `window` but we should migrate to the module style whenever it’s convenient. In this case, go with `handlebars/runtime` if you can; that’s the documented way.

Reply:
Yeah, the `window` hack is a mess but solved the problem of being able to migrate most asset stuff into webpack.  If we can do `import handlebars/runtime`, that'd be great.